docs/UserGuide.adoc

```diff
@@ -209,79 +172,104 @@ Format: `find n\[name]`
 
 Example:
 
-* `find n\Jane doe` +
-Finds a saved contact named Jane Doe.
+* Suppose you want to find Jane doe within NOVA,
```

Comment:
Caps the D for doe.

Reply:
Oh hm for this part I purposely want to make the d as lower case though. Is that ok, or should I change it to "D"?